src/uct/api/uct_def.h

```diff
@@ -299,6 +299,38 @@ typedef ucs_status_t (*uct_sockaddr_conn_request_callback_t)(void *arg,
                                                              size_t length);
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Callback to fill the user's private data on the client side.
+ *
+ * This callback routine will be invoked on the client side before sending the
+ * transport's connection request to the server.
+ * It should be set by the client side when creating an endpoint.
+ * The user's private data should be placed inside the priv_data buffer and it
+ * will be sent to the server side.
```

Comment:
Run on sentence... change to "The users private data should be placed inside the priv_data buffer to be sent to the server side."